addons/website/static/src/js/website.ace.js

```diff
@@ -126,7 +119,7 @@ var ViewEditor = Widget.extend({
                 storeEditorWidth();
             }
         }
-        document.body.addEventListener('mouseup', stopResizing, true);
+        window.addEventListener('mouseup', stopResizing, true);
         self.$('.ace_gutter').mouseup(stopResizing).mousedown(startResizing).click(stopResizing);
```

Comment:
@Gorash  That issue is generating because cursor goes out of the page and event is still attached to `.ace_gutter` so i added event listener to window instead of document.body. And it just solved resizing issue.